packages/create-yoshi-app/__tests__/createApp.test.ts

```diff
@@ -55,6 +55,18 @@ test('it should not create a git repo if the target directory is contained in a
   expect(() => fs.statSync(path.join(projectDir, '.git'))).toThrow();
 });
 
+test('should create a git repo and commit an initial commit by CYA', async () => {
+  (runPrompt as jest.Mock).mockReturnValue(minimalTemplateModel());
+  (verifyRegistry as jest.Mock).mockReturnValue(undefined);
+
+  const tempDir = tempy.directory();
+  const projectDir = path.join(tempDir, 'project');
+  fs.ensureDirSync(projectDir);
+  await createApp({ workingDir: projectDir, install: false, lint: false });
+  const gitLog = execa.sync('git', ['log']).stdout;
```

Comment:
Looking at the last commit message would be a bit cleaner, and you can match the whole message.

```suggestion
  const gitLog = execa.sync('git', ['show', '-s', '--format=%s']).stdout;
```

Reply:
Also, I guess you'd like to run the git command on the [projectDir](https://github.com/sindresorhus/execa#cwd)